lib/qldb.connect.js

```diff
@@ -589,7 +589,9 @@ class qldbConnect {
      */
```

Comment:
Did you see the comment on function buildSqlInsert()? 
        // FIXME Works but creates a timeout
        // if (!this.checkTableExistence(tableName)) {
        //     console.log('table not found and not created');
        // }

Did you manage to solve these timeouts? The mainly occur when using Lambda